### PR TITLE
[14.0][IMP] stock_warehouse_flow: fix is_valid_for_move return value

### PR DESCRIPTION
**Pull request by TDu**
ref: rau-78

### CODE REVIEW

**Review by simahawk — APPROVED** [review]
LG

File: stock_warehouse_flow/models/stock_warehouse_flow.py [hunk 1]
```diff
@@ -339,19 +339,22 @@ def _search_for_move_domain(self, move):
...
     def _search_for_move(self, move):
         """Return the first flow matching the move."""
         domain = self._search_for_move_domain(move)
-        for flow in self.search(domain):
+        flows = self.search(domain)
+        for flow in flows:
             if flow._is_valid_for_move(move):
                 return flow
+        if flows:
+            raise UserError(_("No routing flow available for the conditions"))
```
Comment (sebalix) [inline stock_warehouse_flow/models/stock_warehouse_flow.py]: To help to understand/debugging:
```suggestion
            raise UserError(_("No routing flow available for the move {move} in transfer {picking}.").format(move=move.display_name, picking=move.picking_id.name))
```

**Review by sebalix — COMMENTED** [review]
Just want to update the error message, LG otherwise :)

File: stock_warehouse_flow/models/stock_warehouse_flow.py (hunk 1 above)
Comment (TDu) [reply stock_warehouse_flow/models/stock_warehouse_flow.py]: Changed

**Review by sebalix — APPROVED** [review]

**Comment by sebalix** [comment]
cc @jbaudoux are you OK with this change? So we  can merge.

**Review by jbaudoux — APPROVED** [review]

**Pull request merged**